cmake/add_project_fletch.cmake

```diff
@@ -199,6 +199,11 @@ if( VIAME_ENABLE_FFMPEG )
       ${FLETCH_DEP_FLAGS}
       -DFFmpeg_SELECT_VERSION:STRING=2.6.2
     )
+  elseif( WIN32 )
+    set( FLETCH_DEP_FLAGS
+      ${FLETCH_DEP_FLAGS}
+      -DFFmpeg_SELECT_VERSION:STRING=4.4.1
```

Comment:
Does the VXL ffmpeg plugin still get built with this change?

If you want to proceed with 4.4.1 but I want to continue with 3.3 for now we might just need to parametize this variable (VIAME_FFMPEG_VERSION or the like HL build flag)

Reply:
In my build VXL and ffmpeg are activated, and i can viame_processes_vxl.dll, kwiver_algo_vxl.dll, and kwiver_algo_ffmpeg.dll ..., is that what you mean by "VXL ffmpeg plugin"

Reply:
Not exactly - VXL has it's own ffmpeg reader, which would be in the kwiver_algo_vxl.dll plugin. It gets built whenever VXL is enabled, but if VXL isn't build with ffmpeg support, it will just error out at runtime instead of build time. This has been VIAME's default video reader for a while, but we should switch to the direct ffmpeg wrapper after confirming that there are no regressions with it, or modifications to the frames output from it due to the potential for the frames being slightly temporally off between different readers.

https://github.com/Kitware/kwiver/blob/master/arrows/vxl/vidl_ffmpeg_video_input.h